src/dodal/devices/xbpm_feedback.py

```diff
@@ -22,6 +26,29 @@ def __init__(self, prefix: str = "", name: str = "xbpm_feedback") -> None:
 
     @AsyncStatus.wrap
     async def trigger(self):
-        async for value in observe_value(self.pos_stable):
-            if value:
-                return
+        async def periodic_reminder():
```

Comment:
Could: It would be nice to put this in a separate function in case others find it useful. If you do pull it out it should:
* Go in somewhere like `common/device_utils`
* Have a good docstring describing what it is doing
* Optionally have a nice interface for the caller so that they do not need to worry about how to clean it up e.g.
```python
with periodic_logger("My log message"):
    ...
```
Where the `with` will always clean up the task (see https://docs.python.org/3/library/contextlib.html)

I recommend you give this a go as a learning opportunity, even if we don't end up reusing the logger much.